pip/commands/install.py

```diff
@@ -374,6 +375,24 @@ def run(self, options, args):
                             logger.info(
                                 'Successfully downloaded %s', downloaded
                             )
+                except EnvironmentError as e:
+                    base_msg = (
+                        "Unable to install in installation directory."
+                    )
+                    no_user_msg = (
+                        "Consider using the `--user` option"
+                    )
+                    check_perms_msg = (
+                        "Check the permissions in the installation "
+                        "directory"
+                    )
+                    if e.errno == errno.EPERM:
+                        parts = [check_perms_msg]
+                        if not options.use_user_site:
+                            parts.insert(0, no_user_msg)
+                        logger.error("%s %s", base_msg, " or ".join(parts))
+                    else:
+                        logger.error("%s", base_msg)
```

Comment:
We catch the exception and keep pip running here, I'm not sure that's what we want...